.github/workflows/dev-green.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to Dev-green
+
+on:
+  workflow_dispatch:
+  push:
+    branches: master
+
+concurrency:
+  group: dev-green
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    name: Deploy to Dev-green
+    runs-on: ubuntu-20.04
+    environment: dev
+    env:
+      ECS_CLUSTER: signs-ui
+      ECS_SERVICE: signs-ui-dev-green
+
+    steps:
```

Comment:
Since these steps are (and presumably should be) identical to the other environments, is there an easy way to share the code between them?

Reply:
Oh okay, actually looks like there's a general "deploy to ECS" workflow for CTD that we could use to consolidate things https://github.com/mbta/workflows/blob/main/.github/workflows/deploy-ecs.yml